content/getting-started/linux.md

```diff
@@ -0,0 +1,113 @@
+---
+title: "Linux"
+date: 2019-04-15T11:41:45+02:00
+draft: false
+weight: 1
+---
+
+This page has information on how to install and use TinyGo on Ubuntu, as well as other Linux distros.
+
+If you want to use TinyGo to compile your own or sample code, you can install the release version directly on your machine by following the "Quick Install" instructions below.
+
+You can instead install the full source code to the TinyGo compiler itself, generally for people who wish to contribute to the project or want to build the compiler from sources.
+
+The third option is to use the Docker image. This has the benefit of making no changes to your system, however the image has a large download and installation size.
```

Comment:
Perhaps a link to the docker page would be useful? (on `[Docker image]`).

Reply:
Updated.